pkgs/development/libraries/libsearpc/default.nix

```diff
@@ -9,14 +9,14 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "3.2.0";
+  version = "3.3";
```

Comment:
Just in case, please move python3 from buildInputs to nativeBuildInputs, otherwise cross-compilationn is broken.